mappings/net/minecraft/block/Block.mapping

```diff
@@ -104,6 +104,7 @@ CLASS net/minecraft/class_2248 net/minecraft/block/Block
 		ARG 3 side
 	METHOD method_22358 (Lnet/minecraft/class_2680;Lnet/minecraft/class_3611;)Z
 		ARG 1 state
+	METHOD method_22359 rendersAtFullBrightness (Lnet/minecraft/class_2680;)Z
```

Comment:
Just curious, what is this used for?

Reply:
So magma blocks renders as if it is at lighting level of 15 (full skylight+blocklight) despite the surrounding light level. This is only true for magma blocks. 

This is like the block itself renders at lighting level 15 but has no effect on neighbor blocks.

Reply:
Do you have a better suggestion for the name though, rune?

Reply:
"emissive" is a common name for materials that aren't affected by light (https://www.google.com/search?q=emissive+material) though that might make someone think the block emits light, while it doesn't?

Maybe something like `isEmissiveMaterial` or `usesEmissiveRendering`?

Reply:
Called it `hasEmissiveLighting`.